nncf/quantization/algorithms/hyperparameter_tuner/param_grid.py

```diff
@@ -75,6 +76,10 @@ def _get_minmax_quantization_param_grid() -> ParamGrid:
             RangeEstimatorParameters(min=min_v, max=max_v)
             for min_v, max_v in itertools.product(min_param_values, max_param_values)
         ],
+        "advanced_parameters:propagation_strategy": [
```

Comment:
WHy only two values are checked here?

Reply:
I found these two most useful in terms of accuracy

Reply:
For me it looks strange as I would expect that all options are used. Otherwise why to support them.

Reply:
okay, I can add them if you think that they are useful

Reply:
added